service-workers/header.js

```diff
@@ -0,0 +1,4 @@
+ /* eslint-disable */
```

Comment:
Why the name of this file is header?

Reply:
This is something that goes into the documentation of service-worker-graphql. We should specify each service worker function into their corresponding files (push, activate, fetch, etc). Whatever goes outside those functions is considered header (as workbox on pwa-graphql).

Reply:
Also, I think (as well as on [register.js](https://github.com/vtex/service-worker-graphql/blob/feat/working-sw/node/modules/register.js) that linting this file is unmeaningful, as this will be treated as a string by the service worker merging process.